### PR TITLE
58/Use separate adapters

### PR DESCRIPTION
**Pull request by friedger**
This PR adds adapters for each activity and uses appropriate list item layouts.

It also renames activities and adapters to be consistent across packages:
* ShowBatchesActivity -> BatchesActivity
* BatchesActivity -> BatchDownloadsActivity

Now the activity name contains the model that is queried. The corresponding adapter has the same prefix as the activity. The models are named to distinguish between batches and downloads. The AsyncTask use the corresponding model name as prefix.

### CODE REVIEW

**Comment by hal9002** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://ci.novoda.com/job/download-manager-prb/118/
Test FAILed.

**Comment by hal9002** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://ci.novoda.com/job/download-manager-prb/122/
Test PASSed.

File: demo-extended/src/main/java/com/novoda/downloadmanager/demo/extended/batches/BatchesActivity.java [hunk 1]
```diff
@@ -1,108 +1,91 @@
...
+                    case R.id.show_batches_query_failed_pending:
+                        query = new BatchQuery.Builder().withStatusFilter(DownloadManager.STATUS_FAILED | DownloadManager.STATUS_PENDING).build();
+                        break;
+                    case R.id.show_batches_query_live:
+                        query = new BatchQuery.Builder().withSortByLiveness().build();
+                        break;
+                    case R.id.show_batches_query_deleting:
+                        query = new BatchQuery.Builder().withStatusFilter(DownloadManager.STATUS_DELETING).build();
+                        break;
+                    default:
```
Comment (blundell) [inline demo-extended/src/main/java/com/novoda/downloadmanager/demo/extended/batches/BatchesActivity.java]: don't leave this empty

**Comment by blundell** [comment]
fix the switch later :+1:

**Pull request merged**